src/lib/sub-app/sub-app.factory.ts

```diff
@@ -170,8 +170,9 @@ function updateNpmScripts(
     return;
   }
   const defaultFormatScriptName = 'format';
+  const defaultStartScriptName = 'start:prod';
   const defaultTestScriptName = 'test:e2e';
-  if (!scripts[defaultTestScriptName] && !scripts[defaultFormatScriptName]) {
+  if (!scripts[defaultTestScriptName] && !scripts[defaultFormatScriptName] && !scripts[defaultStartScriptName]) {
```

Comment:
```suggestion
  if (
     !scripts[defaultTestScriptName] && 
     !scripts[defaultFormatScriptName] && 
     !scripts[defaultStartScriptName]
  ) {
```

Reply:
Resolved - Squashed into https://github.com/nestjs/schematics/pull/1169/commits/b8763e0baf932ad40e33de96c407cfde9e85a1b8.

I did run the linter before  originally committing, but maybe there are no line length rules?